### DISCUSSION TITLE
Additional storage location

### DISCUSSION THREAD

**Discussion by Concubine** [AdguardTeam/AdGuardHome#5303] [Ideas]
Hello,

Thanks for a great product, I'm having fun finding out new things about DNS 🥇 

My question is that currently I have very little room on my router with AdGuard installed on it, and was wondering if there is some configuration that would allow to for the Upgrade, Log and Statistics data to be used:

```
Filesystem                Size      Used Available Use% Mounted on
/dev/root                 4.3M      4.3M         0 100% /rom
tmpfs                   248.7M     88.0K    248.6M   0% /tmp
/dev/ubi0_1              55.8M     31.1M     21.8M  59% /overlay
overlayfs:/overlay       55.8M     31.1M     21.8M  59% /
ubi1:syscfg              70.2M    532.0K     66.1M   1% /tmp/syscfg
tmpfs                   512.0K         0    512.0K   0% /dev
**/dev/sda1**                29.2G     24.0K     27.7G   0% /mnt/sda1
```

I would like to direct all of that to /dev/sda1 and other than doing a cron job that will rotate the data was wondering if there is a setting I might have missed that can be configured.

Cheers again for a great product.

UPDATE: 

As a cleaver suggestion by mercygroundabyss over at OpenWRT I mounted my usb stick as /opt and now no more storage issues, this is instead of a complicated solution using cron as I mentioned.

**Comment by Concubine** [comment]
Guys what is this spam, please block this use.

**Comment by ppfeufer** [comment]
In `/opt`:

```
mv AdGuardHome /mnt/sda1/
ln -s /mnt/sda1/AdGuardHome/ .
```

Explanation:
Your AGH directory is now on `/mnt/sda1/` and symlinked to where AGH expects it to be.

Mounting a USB stick as `/opt/` is a bad decision, since AGH is not the only app that might be using `/opt/` and every time you detach the USB stick it's all unmounted and no longer available to the system.
